defaults/main.yml

```diff
@@ -9,108 +9,221 @@
+roundcube__home: '{{ (ansible_local.root.home
+                       if (ansible_local|d() and ansible_local.root|d() and
+                           ansible_local.root.home|d())
+                       else "/var/local") + "/" + roundcube__user }}'
 
-# .. envvar:: roundcube__src
+                                                                   # ]]]
+# .. envvar:: roundcube__comment [[[
 #
-# Base path for git bare repository with Roundcube source
-roundcube__src: '{{ ansible_local.root.src + "/" + roundcube__user }}'
+# The GECOS string set for the Roundcube account.
+roundcube__comment: 'Roundcube Webmail'
 
-
-# .. envvar:: roundcube__www
+                                                                   # ]]]
+# .. envvar:: roundcube__shell [[[
 #
-# Base web root directory for Roundcube website
-roundcube__www: '{{ (ansible_local.nginx.www
-                    if (ansible_local|d() and
-                        ansible_local.nginx|d())
-                    else "/srv/www" ) + "/" + roundcube__user }}'
-
-
-# .. envvar:: roundcube__webserver_user
+# The default shell of the Roundcube account.
+roundcube__shell: '/usr/sbin/nologin'
+                                                                   # ]]]
+                                                                   # ]]]
+# .. Roundcube source and deployment [[[
 #
-# Roundcube webserver user (needs read-only access to the website code)
-roundcube__webserver_user: '{{ ansible_local.nginx.user
-                              if (ansible_local|d() and
-                                  ansible_local.nginx|d() and
-                                  ansible_local.nginx.user|d())
-                              else "www-data" }}'
-
+# -----------------------------------
+#   Roundcube source and deployment
+# -----------------------------------
 
-# .. envvar:: roundcube__git_repo
+# .. envvar:: roundcube__git_repo [[[
 #
 # Roundcube source repository
 roundcube__git_repo: 'https://github.com/roundcube/roundcubemail.git'
 
-
-# .. envvar:: roundcube__git_dest
+                                                                   # ]]]
+# .. envvar:: roundcube__git_dest [[[
 #
 # Roundcube source directory on the host
 roundcube__git_dest: '{{ roundcube__src + "/" + roundcube__git_repo.split("://")[1] }}'
 
-
-# .. envvar:: roundcube__git_tag
+                                                                   # ]]]
+# .. envvar:: roundcube__git_tag [[[
 #
 # Roundcube release tag to deploy
-roundcube__git_version: '1.1.9'
+roundcube__git_version: '1.3.0'
 
-
-# .. envvar:: roundcube__git_checkout
+                                                                   # ]]]
+# .. envvar:: roundcube__git_checkout [[[
 #
 # Default path where Roundcube source files will be deployed
 roundcube__git_checkout: '{{ roundcube__www + "/sites/" +
                              (roundcube__domain if roundcube__domain is string else roundcube__domain[0]) +
                              "/public" }}'
```

Comment:
Perhaps using a domain directly here should be dropped, since the role maintains just one Roundcube install. This would also avoid issues if the `roundcube__domain` value is changed. Multiple Roundcube installations on one host could be handled via internal LXC containers if necessary.